src/services/cube-handler.ts

```diff
@@ -877,8 +879,8 @@ async function setupDimensions(
               SUPPORTED_LOCALES.map((locale) => {
                 const columnName =
                   dimension.metadata.find((info) => info.language === locale)?.name || dimension.factTableColumn;
-                viewSelectStatementsMap.get(locale)?.push(`${dimension.factTableColumn} as "${columnName}"`);
-                rawSelectStatementsMap.get(locale)?.push(`${dimension.factTableColumn} as "${columnName}"`);
+                viewSelectStatementsMap.get(locale)?.push(`"${dimension.factTableColumn}" as "${columnName}"`);
+                rawSelectStatementsMap.get(locale)?.push(`"${dimension.factTableColumn}" as "${columnName}"`);
```

Comment:
If these quote bugs are easy to miss, maybe we create a simple function that handles it and always use that instead? e.g. 

```ts
const colWithAlias = (column: string, alias: string) => {
  return `"${column}" AS "${alias}"`;
}
```

```ts
rawSelectStatementsMap.get(locale)?.push(colWithAlias(dimension.factTableColumn, columnName));
```

Reply:
I've been considering that...  I'd like to tidy up the cube builder when I have some time to spare.  It's a big thing with a lot of reusable functions in it which could help remove duplicate code from dimension and measure processing.  Also I'd like to make it more testable.

As soon as I have some time I'll explore this.